### ISSUE TITLE
Sickrage cannot be installed

### ISSUE THREAD

**Issue by lsantossouza** [htpcBeginner/AtoMiC-ToolKit#91]
Hi

Checking sickrage installation, the installation stops at:

--->Enabling SickRage init.d Autostart at Boot...
cp: cannot stat ‘/home/brazil/.sickrage/init-scripts/init.ubuntu’: No such file or directory

Looking the installer folder, I found the correction:

open the file:

/home/you/AtoMiC-ToolKit/sickrage/sickrage-constants.sh

change 

APPINITDLOC=$APPPATH'/init-scripts/init.ubuntu'

to:
APPINITDLOC=$APPPATH'/runscripts/init.ubuntu'

So, restart the process and you will install the sicrage.

Have a nice day!!!

**Comment by TommyE123** [comment]
Hi, 

Firstly thanks ever so much for logging an issue! It can be a lonely job sometimes writing these scripts thinking no one is using them!!!

I took a look at the error you got and your suggestion and your quite right it was wrong so apologies for that. However last night I rewrote all the scripts for SickRage before you logged this issue and tested them all against a Raspbian and Ubuntu environment using system and init.d. This problem was cleaned up whilst doing that process.

If you update the toolkit via the maintenance menu option everything should be ok now.
As a side note I'd suggest backing up and reinstalling SickRage so it uses the newer configuration.

Thanks 
Tom

**Comment by lsantossouza** [comment]
Hi Tom

I found your script in 2016 when I was looking for a definitive way to install the sickrage and others at Cubietruck.  Your script was the best way to install the sickrage in my cubietruck. 

I follow your site and I checked that you dont update yours articles and I tought that you quite to post.

So, how your script is very well, the way to say thank you, was update your script and leave a clue for someone update too. 

A question, when you will update your script to install Plex automatically?

More one time, thank you for your project and If have some thing to help, let me known. Is not so good ride alone. 


Best Regards

Luis Souza
